surface/include/pcl/surface/texture_mapping.h

```diff
@@ -64,16 +64,15 @@ namespace pcl
       */
     struct Camera
     {
-      Camera () : focal_length (), focal_length_w (-1), focal_length_h (-1),
-        center_w (-1), center_h (-1), height (), width () {}
+      Camera () = default;
       Eigen::Affine3f pose;
-      double focal_length;
-      double focal_length_w;  // optional
-      double focal_length_h;  // optinoal
-      double center_w;  // optional
-      double center_h;  // optional
-      double height;
-      double width;
+      double focal_length{0};
```

Comment:
All double values - minor nitpick.
```suggestion
      double focal_length{0.0};
```